mobile-app/app/screens/AppNavigator/screens/Portfolio/components/PortfolioCard.tsx

```diff
@@ -158,12 +167,18 @@ function getEmptyScreenDetails(type?: ButtonGroupTabKey): {
     case ButtonGroupTabKey.AllTokens:
     default:
       return {
-        icon: EmptyPortfolioIcon,
+        icon: domain === "DFI" ? EmptyPortfolioIcon : EmptyEvmPortfolioIcon,
```

Comment:
I'm guessing for EVM this empty portfolio should only be displayed when DFI for EVM balance === 0. But let's wait for confirmation

Reply:
yess!! Will add an additional condition of `DFI for EVM balance === 0` once transfer domain is up!